build.xml

```diff
@@ -21,23 +21,17 @@
         <delete dir="${output.dir}" />
         <mkdir dir="${output.dir}" />
 
-        <!-- Check for updates to Composer -->
-        <exec executable="php" failonerror="true">
-            <arg value="composer.phar"/>
-            <arg value="self-update"/>
-        </exec>
-
         <!-- Install dependencies through Composer -->
         <exec executable="php" failonerror="true">
             <arg value="composer.phar"/>
             <arg value="install"/>
         </exec>
 
         <!-- Update dependencies through Composer -->
-        <exec executable="php" failonerror="true">
+        <!-- <exec executable="php" failonerror="true">
             <arg value="composer.phar"/>
             <arg value="update"/>
-        </exec>
+        </exec> -->
```

Comment:
Given that we don't include `composer.lock` (it's a library so it's correct) we don't need to update so I'm happy with removing this.